addons/account_payment/views/account_portal_templates.xml

```diff
@@ -1,12 +1,58 @@
 <odoo>
+    <template id="portal_my_home_overdue_invoice" name="Portal layout : overdue invoices" inherit_id="portal.portal_breadcrumbs" priority="30">
+        <xpath expr="//ol[hasclass('o_portal_submenu')]" position="inside">
+            <li t-if="page_name == 'overdue_invoices'" t-attf-class="breadcrumb-item active">
+                <a t-attf-href="/my/invoices?{{ keep_query() }}">Invoices &amp; Bills</a>
+            </li>
+            <li t-if="payment" class="breadcrumb-item active">
+                <t t-out="payment['reference']"/>
+            </li>
+            <li t-elif="page_name == 'overdue_invoices'" class="breadcrumb-item active">
+                <em>overdue invoices</em>
+            </li>
+        </xpath>
+
+        <xpath expr="//ol[hasclass('o_portal_submenu')]" position="after">
+            <a t-if="page_name == 'invoice' and not invoice" class='btn btn-primary' href='/my/invoices/overdue'>Pay overdue</a>
+        </xpath>
+    </template>
+
+    <template id="portal_overdue_invoices_page" name="Overdue payments">
+        <t t-call="portal.portal_layout">
+            <t t-if="payment">
+                <div class="row bg-light mb-4 rounded p-5">
+                    <div class="col-6">Amount</div>
+                    <div class="col-6">Reference</div>
+                    <h4 class="col-6">
+                        <span t-out="abs(payment['amount'])"
+                            t-options='{"widget": "monetary", "display_currency": payment["currency"]}'/>
+                    </h4>
+                    <h4 class="col-6">
+                        <span t-esc="payment['reference']"/>
+                    </h4>
+                </div>
+                <t t-call="payment.form"/>
+            </t>
+            <t t-else="">
+                <div class="row bg-light mb-4 rounded p-5">
+                    <h4>You don't have any overdue invoices!!</h4>
```

Comment:
There are many payment flows that display a recap of the payment on a landing page. For example, try generating a payment link (`.../payment/pay?...`) from an invoice and pay from there. Here you could reuse the overdue invoices page and display the recap instead of the h4.

Reply:
I asked the PO, and he said I can redirect to the invoices page

Reply:
Comment resolved but it's still there.